resources/js/features/root/pages/AdminUsersList.page.tsx

```diff
@@ -60,21 +59,17 @@ export default (() => {
           "id",
           "name",
           "email",
-          "hasPassword",
-          "createdAt",
-          "facilitiesMember",
-          "numFacilities",
           "hasGlobalAdmin",
-        ]}
-        initialVisibleColumns={[
-          "name",
-          "email",
-          "hasPassword",
           "createdAt",
-          "facilitiesMember",
-          "hasGlobalAdmin",
+          "createdBy.name",
+          "updatedAt",
+          "hasEmailVerified",
+          "hasPassword",
```

Comment:
To jest akurat dość ważna kolumna, więc ja bym dał email, hasEmailVerified, hasPassword, passwordExpireAt, tam zaraz za name. No ale to detal a jak jeszcze będziesz ten mechanizm przepisywał to można wtedy o tym pogadać.

Reply:
Ok